package.json

```diff
@@ -79,7 +79,6 @@
     "gcp-metadata": "^0.9.0",
     "google-auth-library": "^2.0.0",
     "google-gax": "^0.20.0",
-    "google-proto-files": "^0.17.0",
```

Comment:
Is this related at all to #133?

Reply:
It isn't (that I know of). This package is used by gax, but just kinda hanging out here as far as I can tell.